api/v2/openapi.yaml

```diff
@@ -496,6 +557,111 @@ components:
           maxItems: 100
           items:
             $ref: '#/components/schemas/Location'
+    Badge:
+      title: Badge
+      description: Badges for certain actions
+      type: string
+      nullable: true
+      enum:
+        - next_level
+        - next_level_class
+        - report
+        - new_report
+        - two_shops
+        - shop_first_report
+        - week_1
+        - week_3
+        - week_full
+        - week_special
+        - trusted_shop
+        - trusted_user
+    NewBadge:
+      title: NewBadge
+      description: latest Badges of a specific user
+      type: object
+      nullable: false
+      required:
+        - badges
+        - reason
+      properties:
+        badges:
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/Badge'
+        reason:
+          type: string
+          example: occupancy
+    AllBadges:
+      title: AllBadges
+      description: all Badges of a specific user plus the total ammount of badges
+      type: object
+      nullable: false
+      required:
+        - badgeAndCounter
+        - countAllBadges
+      properties:
+        BadgeAndCounter:
```

Comment:
Properties werden in `lowerCamelCase` geschrieben. Und `badges` wäre hier ausreichend, sonst sind die Namen irgendwann überladen.

Reply:
ok